src/Process/GlobeTileProcessing.js

```diff
@@ -70,10 +72,13 @@ export function globeCulling(minLevelForHorizonCulling) {
 }
 
 function computeNodeSSE(camera, node) {
+    const v = new THREE.Vector3();
+    v.setFromMatrixScale(node.matrixWorld);
     const boundingSphere = node.geometry.boundingSphere;
     const distance = Math.max(
         0.0,
-        camera.camera3D.position.distanceTo(node.centerSphere) - boundingSphere.radius);
+        camera.camera3D.position.distanceTo(
```

Comment:
Why don't take into account the rotations and translations inherited by the parents?